client/src/pages/LoginForLandingpage.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react';
+import Typography from '@material-ui/core/Typography';
+import { Link } from 'react-router-dom';
+import { Grid, makeStyles, Button, CircularProgress } from '@material-ui/core';
+import { useMutation } from '@apollo/react-hooks';
+import gql from 'graphql-tag';
+import Alert from '@material-ui/lab/Alert';
+import { CustomTextField } from '../components/CustomTextField';
+
+const useStyles = makeStyles((theme) => ({
+	paper: {
+		display: 'flex',
+		flexDirection: 'column',
+		alignItems: 'center',
+		justifyContent: 'center',
+		minHeight: '100vh',
+		marginLeft: theme.spacing(7),
+		marginRight: theme.spacing(7),
+
+	},
+	margin: {
+		marginTop: theme.spacing(4),
+	},
+	roundButton: {
+		borderRadius: '20px',
+		maxWidth: '300px',
+		fontWeight: 700,
+	},
+	alert: {
+		marginTop: '20px',
+	},
+	label: {
+		marginBottom: theme.spacing(4),
+	},
+	input: {
+		minWidth: '300px',
+	}
+}));
+
+const LOGIN_USER_MUTATION = gql`
+	mutation login($username: String!, $password: String!) {
+		login(username: $username, password: $password) {
+			id
+			email
+			username
+			createdAt
+			token
+		}
+	}
+`;
```

Comment:
I would move styles and mutation to separated files, because this component growth too much